AI_DEV_POLICY.md

```diff
@@ -0,0 +1,317 @@
+   - Thoroughly review all AI-generated code before submission
+   - Ensure you understand what the code does and how it works
+   - Be able to explain and defend the implementation in code reviews
+   - You take responsibility for the contribution's content and origin
+
+2. **Verify License Compliance**
+   - Ensure AI tool terms and conditions do not conflict with:
+     - Apache 2.0 license
+     - Kiali's intellectual property policies
+     - The [Open Source Definition](https://opensource.org/osd/)
+   - Check if the AI tool's output restrictions are compatible with open source distribution
+
+3. **Address Third-Party Materials**
+   - If AI output includes pre-existing copyrighted materials or code from third parties:
+     - Verify you have permission to use and modify such materials
+     - Ensure the third-party license is compatible with Apache 2.0
+     - Provide proper attribution and license information
+   - Consider using AI tool features that suppress or flag responses similar to copyrighted materials
+
+4. **Test Thoroughly**
+   - Write and run appropriate tests for AI-assisted code
+   - Verify the code works as intended in the Kiali environment
+   - Test edge cases and error conditions
+   - Ensure integration with existing Kiali components
+
+5. **Disclose AI Assistance**
+   - Contributors are **REQUIRED** to disclose when AI assistance was used to generate issues, pull requests, and commits
+   - In commit messages or pull request description fields, identify the code assistant that you used
+   - Use a trailer such as `Assisted-by:` or `Generated-by:`
+   - Example for commits and PRs:
+     ```
+     Assisted-by: Claude Code
+     ```
+   - In source file comments, indicate the use of the code assistant
+   - Example for source files:
+     ```
+     # Generated by Cursor
+     ```
+     or
+     ```
+     // Assisted-by: Claude Code
+     ```
+   - Be transparent and honest about your development process
+   - Maintainers may request additional information about code origin during reviews
+
+### Quality Standards
+
+All contributions, whether AI-assisted or not, must meet these standards:
+
+1. **Technical Depth**
+   - Demonstrate understanding of the problem space
+   - Show awareness of Kiali architecture and design patterns
+   - Consider performance, scalability, and maintainability
+
+2. **Code Quality**
+   - Follow Kiali coding standards and conventions
+   - Include appropriate error handling
+   - Provide meaningful variable and function names
+   - Avoid overly complex or obscure implementations
+
+3. **Documentation**
+   - Include clear comments for complex logic
+   - Update relevant documentation
+   - Provide examples where appropriate
+   - Document any assumptions or limitations
+
+4. **Security**
+   - Avoid common security vulnerabilities (injection, XSS, etc.)
+   - Follow secure coding practices
+   - Handle sensitive data appropriately
+   - Consider security implications of dependencies
```

Comment:
I'd remove this bloat, and if necessary move it to https://github.com/kiali/kiali/blob/master/CONTRIBUTING.md

Reply:
i'll remove